### ISSUE TITLE
Some errors are omitted for multi-statement lines

### ISSUE THREAD

**Issue by weirdan** [vimeo/psalm#1321]
Comparing https://getpsalm.org/r/de1e470acf to https://getpsalm.org/r/6acd309147 (where the only difference is that the former has 4 `strlen`s each on its own line, and the latter has them on the same line) Psalm emits different number of issues (4 vs 2). Every second issue is omitted.

**Expected**: 4 issues with either code snippet (or no issues when #1320 is fixed)
**Actual**: 2 issues reported, 2 skipped

For (https://getpsalm.org/r/b7041fb172)
```php
<?php

  $a = $b = $c = $d = rand(0,1) ? [1] : 'a';

  strlen($a);  strlen($b);  strlen($c);   strlen($d);
```
only one issue is reported, even though there are 4 problematic calls.

**Comment by muglug** [comment]
Closing as not a bug - I deduplicate error messages s in Psalm's output, and this error is exactly the same

**Comment by weirdan** [comment]
Ah, ok.
